TrashMob/client-app/src/components/VolunteerOpportunities.tsx

```diff
@@ -28,24 +28,19 @@ export const VolunteerOpportunities: React.FC = () => {
         <>
             <HeroSection Title='Recruiting' Description='We’d love to have you join us.' />
             <Container>
-                <Row>
-                    <Col md={8} className='mb-2'>
+                <Row className='my-4'>
+                    <Col md={6} className='mb-2 text-center'>
```

Comment:
Col md={6} is too small for the whole text + number
![image](https://github.com/user-attachments/assets/9beb1051-1dc5-41bb-8387-c326b2f09745)

Reply:
Fixed, updated to 8, got rid of the placeholder col that was there from before, now the line doesn't wrap like in your screenshot.